geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStore.scala

```diff
@@ -48,23 +49,12 @@ class HBaseDataStore(val connection: Connection, override val config: HBaseDataS
 
   override val adapter: HBaseIndexAdapter = new HBaseIndexAdapter(this)
 
-  override val stats: GeoMesaStats =
-    if (config.remoteFilter) { new DistributedRunnableStats(this) } else { new UnoptimizedRunnableStats(this) }
+  override val stats: GeoMesaStats = new RunnableStats(this)
 
   // zookeeper locking
   override protected val mock: Boolean = false
   override protected val zookeepers: String = ZKConfig.getZKQuorumServersString(connection.getConfiguration)
 
-  override def delete(): Unit = {
```

Comment:
this is implemented already in the super class, and this method doesn't do anything different so was removed